dev/com.ibm.ws.security.jaspic_fat/fat/src/com/ibm/ws/security/jaspic11/fat/audit/JASPIFormLoginJACCAuthorizationAuditTest.java

```diff
@@ -74,6 +79,10 @@
 @Mode(TestMode.FULL)
 public class JASPIFormLoginJACCAuthorizationAuditTest extends JASPITestBase {
 
+    @ClassRule
+    public static RepeatTests r = EERepeatActions.repeat("com.ibm.ws.security.jaspic11.fat.audit",
+    EERepeatActions.EE11).with(new EmptyAction().fullFATOnly());
```

Comment:
```suggestion
    /**
     * Need the first repeat to make sure that audit-2.0 from a previous repeat gets put back to audit-1.0
     */
    @ClassRule
    public static RepeatTests auditRepeat = RepeatTests.with(new FeatureReplacementAction("audit-2.0", "audit-1.0"))
                    .andWith(new FeatureReplacementAction("audit-1.0", "audit-2.0"));
```
Don't need the EE repeat since it is already done in FATSuite.